src/lib/Server/Controller/Role.php

```diff
@@ -846,10 +758,16 @@ public function loadRoleAssignmentForUserGroup($groupPath, $roleId, Request $req
      */
     public function listPoliciesForUser(Request $request)
     {
+        $user = $this->userService->loadUser($request->query->get('userId'));
```

Comment:
Probably:
```suggestion
        $user = $this->userService->loadUser((int)$request->query->get('userId'));
```
right?

Reply:
Prolly yes, but `loadUser` is still not typehinted, and there are places like this all along the code, so they should be also be fixed in some other PR.

I am also not sure, why this param is not included in url and then used as an attribute for this method.